sbt-version-policy/src/main/scala/sbtversionpolicy/SbtVersionPolicySettings.scala

```diff
@@ -125,8 +126,9 @@ object SbtVersionPolicySettings {
       val updateConfig = versionPolicyUpdateConfiguration.value
       val warningConfig = versionPolicyUnresolvedWarningConfiguration.value
       val excludedModules = ignoredModulesOfCurrentBuild.value
+      val extractVersion = versionPolicyModuleVersionExtractor.value
```

Comment:
In case you would go for a `PartialFunction`, here you could define `extractVersion` with something like the following:

~~~ scala
val extractVersion = versionPolicyModuleVersionExtractor.value.orElse(_.revision)
~~~